aep/general/0300/aep.md.j2

```diff
@@ -0,0 +1,98 @@
+# AEP Editions
+
+This AEP describes how the AEP specification is versioned.
+
+## Edition based versioning
+
+AEPs will be published in editions, similar but not identical to specifications
+such as those for the C++ or Rust programming language:
+
+- The specification will not change for a non-preview edition after it is
+  published.
+- The naming of the edition will be based on the year in which it is published
+  (e.g. aep-2025).
+- Upon publishing a non-preview edition, the current prevision will be removed,
+  and the next preview edition will be created (e.g. aep-2027-preview). This
+  preview edition will accumulate changes until the next edition is published.
+- AEP editions **MAY** have patch version updates which adhere to semantic
+  versioning (e.g. aep-2025.25).
+  - The patch version will only contain fixes to typos and clarification on
+    existing guidance.
+- Editions will be published every 2 years.
+
+## Breaking changes between editions
```

Comment:
I think we should include the rationale we developed about why the vast majority of substantive changes are breaking.  In particular, the point that if we introduce a new pattern, and say "if you're doing this pattern, do it like this", previously compliant APIs that solve that pattern in a different way would become non-compliant.